.github/workflows/publish.yml

```diff
@@ -2,7 +2,7 @@ name: Release
 on: [workflow_dispatch] # Manual trigger
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Hm, despite this being 100% a change that I'd do, I'm conflicted between this and using `ubuntu-latest` for future-proofing, even though iirc, it's still at 20.04

Reply:
GitHub's team still relays on 20.04 LTS when using the "latest" term - so i just thought of upstreaming this if this is even okay for this including rest of libraries that i planned to PR.

Reply:
...actually, let's keep it at 22.04 and then move to latest once the rollout is done

Reply:
Alright i gotcha.